public/index.js

```diff
@@ -1,7 +1,14 @@
-var React = window.React;
-var ReactDOM = window.ReactDOM;
+window.oc = window.oc || {};
+oc.cmd = oc.cmd || [];
 
-ReactDOM.render(
-  React.createElement('h1', null, 'Hello World!'),
-  document.getElementById('root')
-);
+oc.cmd.push(function() {
+
+	oc.require('React', '//cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react.min.js', function(){
+		oc.require('ReactDOM', '//cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-dom.min.js', function(){
```

Comment:
This will just execute the callback straightaway if window.React and window.ReactDOM is there